### PR TITLE
Revert to previous version of react-focus-lock [SATURN-1191]

### PR DESCRIPTION
**Pull request by petesantos**
Version > 2.1.0 of react-focus lock seems to have introduced a backwards incompatibility. I am working on identifying and providing repro steps to the author

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -18734,9 +18599,9 @@
       },
       "dependencies": {
         "@babel/runtime": {
-          "version": "7.6.2",
-          "resolved": "https://registry.npmjs.org/@babel/runtime/-/runtime-7.6.2.tgz",
-          "integrity": "sha512-EXxN64agfUqqIGeEjI5dL5z0Sw0ZwWo1mLTi4mQowCZ42O59b7DRpZAnTC6OqdF28wMBMFKNb/4uFGrVaigSpg==",
+          "version": "7.6.3",
```
Comment (panentheos) [inline package-lock.json]: This seems like it upgrades a lot of unrelated stuff. Can we double check that?

**Review by panentheos — APPROVED** [review]
Approach is good, but see comment.

File: package-lock.json (hunk 1 above)
Comment (petesantos) [reply package-lock.json]: I corrected this. I should have checked the lock diff before submitting the PR - thanks!
